modules/modal/react/lib/Modal.tsx

```diff
@@ -41,6 +66,47 @@ const Container = styled('div')({
   animationDuration: '0.3s',
 });
 
+const transformOrigin = {
+  horizontal: 'center',
+  vertical: 'top',
+} as const;
+
+function onInitialFocus(
+  modalEl: HTMLElement | null,
+  firstFocusEl: HTMLElement | null = null
+): HTMLElement {
+  if (firstFocusEl) {
+    return firstFocusEl;
+  } else {
+    const firstFocusable = modalEl && modalEl.querySelector<HTMLElement>('[aria-label=Close],h3');
+    if (firstFocusable) {
+      if (firstFocusable.tagName === 'H3') {
+        // If there is no close icon, we need to transfer focus to the header.
+        // Setting tabIndex allows the header to be focusable.
+        // We do the header instead of the next focusable element to prevent useful context from being skipped
+        firstFocusable.tabIndex = 0;
```

Comment:
Can you add `&:focus { outline: 'none' }` to the card title in `Card`? I believe we discussed hiding the focus outline if the heading receives focus with a11y.

Reply:
That's not handled by the `InputProvider`?

Reply:
No, `InputProvider` just wraps everything with a div that indicates the input type. It does not provide any overriding styling to avoid being too obtrusive. It's up to each individual component to manage their own styling based off of the data attrs it provides.

In this case I think we want to hide the focus outlines at all times, so it shouldn't really be dependent on the `InputProvider` state anyway.

Reply:
Is putting `&:focus { outline: 'none';  }` a leak if it goes in `Card`? The `Card` component doesn't necessarily know focus is possible on the `header`. I can add a `firstFocusable.style.outline = 'none';`. That is more DOM manipulation, but this parent knows about what is happening in the child rather than the child needing to know what parents might be doing.

Reply:
I've added `firstFocusable.style.outline = 'none';` below this line.